set path guidance

spdegabrielle opened the pull request:
resolve https://github.com/racket/racket-lang-org/issues/101 ?

spdegabrielle commented:
Please squash commits

sorawee commented on pkgs/racket-doc/scribblings/more/more.scrbl:
```diff
@@ -95,6 +95,8 @@ start @exec{racket} with no command-line arguments:
   > 
 }
 
+@margin-note*{Set your @tt{PATH} environment variable so you can use raco and other Racket command line functions. On macOS:@tt{sudo sh -c 'echo "/Applications/Racket v7.7/bin" >> /etc/paths.d/racket'}. On Windows add the racket @tt{bin} path to @tt{Path} in 'Environment Variables' (under System Properties, Advanced tab)}
```
1. I think it's better to use `margin-note` rather than `margin-note*`
2. Use `@exec{raco}`.
3. I think Racket people prefer MacOS over macOS
4. Add a space after `macOS:`
5. Add a `:` after `Windows`.
6. Users might not know that they need to change `7.7` to the current version of Racket. We also shouldn't hard code a version number since we need to change it every release. I'm not sure what placeholder we should put there.

samth replied:
You can produce the current version number with the `version` function.

sorawee replied:
I was hoping for something like `echo "/Applications/Racket v<racket version number>/bin"` with a different font for `<racket version number>` to make it clear that it's a placeholder that users need to fill in. But I guess the `version` function is a simpler solution.

rfindler replied:
Perhaps also worth noting: the program isn't guaranteed to be in /Applications, even if people installed one of the standard releases. They are encouraged to drag it there, but they might drag it to the Desktop or they might just run it on the mounted dmg (I know the latter is a real thing as there was a bug that affected only people like that and we got reports....took a while to figure out that the dmg thing was relevant!)

spdegabrielle replied:
Made all suggested changes. Please review and let me know if you see any issues.

spdegabrielle commented:
Please *Squash* if you are happy.

pmatos commented on pkgs/racket-doc/scribblings/more/more.scrbl:
```diff
@@ -95,6 +95,8 @@ start @exec{racket} with no command-line arguments:
   > 
 }
 
+@margin-note{Set your @tt{PATH} environment variable so you can use @exec{raco} and other Racket command line functions. On MacOS: @tt{sudo sh -c 'echo "/Applications/Racket v@version{}/bin" >> /etc/paths.d/racket'} (This assumes you have installed Racket in the @tt{Applications} folder). On Windows: add the racket @tt{bin} path to @tt{Path} in 'Environment Variables' (under System Properties, Ad ...
```
Should we wrap this?

spdegabrielle replied:
what is the etiquette here? Can I click 'Resolve conversation' or does it need to be a repo owner?

spdegabrielle replied:
![image](https://user-images.githubusercontent.com/108047/83858759-def4ef80-a714-11ea-9949-de571b2e3e0a.png)

sorawee replied:
I think what @pmatos meant is that the Scribble source should be broken up to several lines, not one long line.

spdegabrielle replied:
wrap fixed
![image](https://user-images.githubusercontent.com/108047/83862849-698c1d80-a71a-11ea-9a16-eeedacd3a3f1.png)

still renders OK
![image](https://user-images.githubusercontent.com/108047/83862889-7a3c9380-a71a-11ea-98e2-2fa7fd0fc534.png)

spdegabrielle commented:
Hi

I think I have addressed comments by @sorawee @samth and @rfindler 

Please let me know if there are any issues blocking this PR.

![image](https://user-images.githubusercontent.com/108047/83858808-f03dfc00-a714-11ea-91f8-dc74fd03e001.png)

sorawee commented:
The single quote is supposed to be straight, but it's not. If anyone copies the command and uses it without correction, it won't work. I think you can use [`literal`](https://docs.racket-lang.org/scribble/base.html?q=literal#%28def._%28%28lib._scribble%2Fbase..rkt%29._literal%29%29) to fix the problem.

spdegabrielle commented:
Thanks @sorawee that fixed the problem; 
![image](https://user-images.githubusercontent.com/108047/83862243-925fe300-a719-11ea-9cf9-58e90919e9a1.png)

OK now?

sorawee commented:
Sorry, I just saw that there's also another wrong shape of quote at `Environment Variables`. The opening one should be backtick.

spdegabrielle commented:
wrap in code fixed.
![image](https://user-images.githubusercontent.com/108047/83863032-a9530500-a71a-11ea-99b7-74b705e8a7c9.png)

spdegabrielle commented:
> Sorry, I just saw that there's also another wrong shape of quote at `Environment Variables`. The opening one should be backtick.

I have revitalised with a cuppa and a jammy dodger 
![IMG_0752](https://user-images.githubusercontent.com/108047/83866470-c3431680-a71f-11ea-934e-63eb9866f6c2.jpg)
and changed the quote to `
![image](https://user-images.githubusercontent.com/108047/83866604-eff72e00-a71f-11ea-971e-7d5c0a7097c2.png)

mflatt commented:
A few more tweaks? "MacOS" should be "Mac OS" (at least by the Racket documentation rules). For the Windows part, I'd use `@onscreen` for "Path", "Environment Variables", "System Properties", and "Advanced". (But if you're tired of all these little changes, I'd be happy to make adjustments after merging.)

spdegabrielle commented:
No problem. How does this look?

![image](https://user-images.githubusercontent.com/108047/83877178-fa222800-a731-11ea-9e79-3d15de84c77e.png)

mflatt commented:
The quotes around "Environment Variables" now seem redundant to me.

Somewhat more substantively: Do we want to recommend `>>` or `>` for `/etc/paths.d/racket`? It seems like `>` has the advantage that it will replace a path for an older installation, but maybe you had something specific in mind with `>>`.

sorawee commented:
I think `>>` is better. It doesn't delete any data.

spdegabrielle commented:
> The quotes around "Environment Variables" now seem redundant to me.
fixed 

I would use `>` but I followed the example in Beautiful Racket.

![image](https://user-images.githubusercontent.com/108047/83878745-833a5e80-a734-11ea-811c-a7761d883553.png)

spdegabrielle commented:
https://beautifulracket.com/setting-the-mac-os-path.html

mflatt commented:
Ok - unless there are further comments, I'll merge this soon.
